hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -1513,9 +1513,14 @@ void
 vlapic_reset(struct vlapic *vlapic)
 {
 	struct lapic *lapic;
+	void *apic_page;
 
 	lapic = vlapic->apic_page;
-	memset(lapic, 0, sizeof(struct lapic));
+
+	apic_page = (void *)vlapic->apic_page;
+	memset(apic_page, 0, CPU_PAGE_SIZE);
+	if (vlapic->pir_desc)
+		memset(vlapic->pir_desc, 0, sizeof(struct pir_desc));
```

Comment:
This is different with what you submitted in mailinglist. This one is more old. Please update

Reply:
Oops. I applied the V1 when adding Acked-by. Will fix it with new PR.

Reply:
Applied the correct patch which match the one acked on mail list.